components/TaskWrapper.vue

```diff
@@ -0,0 +1,82 @@
+<template>
+  <section>
+    <div class="task-area">
+      <AddTaskCard />
+      <Todo v-for="todo in computedList" :key="todo.id" :todoItem="todo" />
+    </div>
+
+    <div class="load-btn">
+      <button v-if="loadMore || showLess" @click="onLoadBtnClick">
+        {{ btnText }}
+      </button>
+    </div>
+  </section>
+</template>
+
+<script>
+import { mapState } from "vuex";
+export default {
+  data() {
+    return {
+      firstIndex: 0,
+      lastIndex: 0,
+    };
+  },
+  computed: {
+    ...mapState("todo", ["todoList", "limit"]),
+    btnText() {
+      if (this.loadMore) {
+        return `${this.$t("Load")} ${this.$t("More")}`
+      } else if (this.showLess) {
+        return `${this.$t("Show")} ${this.$t("Less")}`
+      } else {
+        return "";
+      }
+    },
+    loadMore() {
+      if (this.todoList.length > this.computedList.length) {
+        return true;
+      } else {
+        return false;
+      }
+    },
+    showLess() {
+      if (this.computedList.length > this.limit) {
+        return true;
+      } else {
+        return false;
+      }
+    },
```

Comment:
Computed properties must be prefixed with can/is/have depending on the context.

Also we can avoid if else by using an inline version of it.
```suggestion
    isShowLessState() {
     return this.computedList.length > this.limit
    },
```

Reply:
resolved